src/imageio.cs

```diff
@@ -2356,4 +2368,17 @@ interface CGImageAuxiliaryDataInfo {
 		NSData Data { get; set; }
 		NSDictionary DataDescription { get; set; }
 	}
+
+	[Partial]
+	[Mac (10, 15), iOS (13, 0), TV (13, 0), Watch (6, 0)]
+	interface CGImageAnimationOptions {
+		[Field ("kCGImageAnimationDelayTime")]
+		NSString DelayTime { get; }
+
+		[Field ("kCGImageAnimationLoopCount")]
+		NSString LoopCount { get; }
+
+		[Field ("kCGImageAnimationStartIndex")]
+		NSString StartIndex { get; }
```

Comment:
Is StartIndex an NSNumber? Could we use BindAs?

Reply:
From Apple documentation: `const CFStringRef kCGImageAnimationStartIndex;`

I think this means that we should leave it as an NSString?

Reply:
From the headers:
```
/* Starts the animation at the given index. Defaults to 0
+ * Value is a CFNumber
+ */
+IMAGEIO_EXTERN const CFStringRef kCGImageAnimationStartIndex  IMAGEIO_AVAILABLE_STARTING(10.15, 13.0);
```
So it is a CFNumber.

Reply:
Good catch :) Changed to `NSNumber`

Reply:
ditto, stupid manuel